Taxi/Taxi/Presenter/AddTaxiParty/InfoContainer.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  InfoContainer.swift
+//  Taxi
+//
+//  Created by JongHo Park on 2022/06/14.
+//
+
+import SwiftUI
+
+// MARK: - 정보 선택 영역 가이드 뷰
+extension AddTaxiPartyView {
+    struct InfoContainer: ViewModifier {
+        let title: String
+        let selectedInfo: String
+        let toggle: Bool
+        let onClick: () -> Void
+
+        func body(content: Content) -> some View {
+            VStack(alignment: .leading, spacing: 0) {
+                section
+                    .zIndex(2)
+                content
+                    .zIndex(1)
+                    .frame(height: toggle ? nil : 0)
+                    .contentShape(Rectangle())
+                    .clipped()
+            }
+        }
+
+        private var section: some View {
+            Button {
+                onClick()
+            } label: {
+                HStack {
+                    Text(title)
+                        .font(.system(size: 18, weight: toggle ? .bold: .medium, design: .default))
+                        .foregroundColor(toggle ? Color.customBlack : .darkGray)
+                        .lineLimit(1)
+                    Spacer()
+
+                    Text(selectedInfo)
+                        .font(Font.custom("AppleSDGothicNeo-Medium", size: 18))
```

Comment:
저희 기본 폰트로 바꾸지 않았나요? 어차피 텍스트 스타일 적용하겠지만 애플고딕의 잔재가...!!
```suggestion
                        .font(.system(size: 18))
```

Reply:
엥 전체 텍스트 스타일이 아예 바뀐거였나요? 전 애니메이션 때문에 저 부분만 바뀐 줄 알았는데

Reply:
그렇다네요 ㅎㅎ 제가 잘못 알고 있었던걸로